.github/workflows/crowdin-source-updater.yml

```diff
@@ -0,0 +1,61 @@
+name: "Crowdin Action"
+
+# **What it does**: Download new source from crowdin project.
+# **Why we have it**: We want to be able to pull-update base (EN) translations to the project.
+# **Who does it impact**: Everyone collaborating on the project, and the translators.
+# For more info: https://github.com/crowdin/github-action/blob/master/action.yml
+
+on:
+    push:
+        branches: [ "chore/crowdin-source-updater" ] # Delete after tst.
+    workflow_dispatch:
```

Comment:
Should we remove or is it needed for the GH action to run?

Reply:
Yes we need the `workflow_dispatch` so we can trigger it manually, I removed the push block tho, the test ran successfully

Reply:
> so we can trigger it manually

Who, exactly, and how? We, developers, by hitting the button in `Actions`, or we have some more stuff going on?